linera-views/src/views/bucket_queue_view.rs

```diff
@@ -141,14 +141,15 @@ pub struct BucketQueueView<C, T, const N: usize> {
     delete_storage_first: bool,
 }
 
-impl<C, T, const N: usize> View<C> for BucketQueueView<C, T, N>
+impl<C, T, const N: usize> View for BucketQueueView<C, T, N>
 where
-    C: Context + Send + Sync,
-    ViewError: From<C::Error>,
+    C: Context,
     T: Send + Sync + Clone + Serialize + DeserializeOwned,
 {
     const NUM_INIT_KEYS: usize = 2;
 
+    type Context = C;
```

Comment:
It's both type parameter (`BucketQueueView<C,..>`) and type member. It should probably be one of the two, not both.

Reply:
What I'd really like to do is have the view not depend on its context type at all, but that's not trivial and would make this PR much bigger.